README.md

```diff
@@ -237,7 +237,48 @@ methods:{
 }
 ```
 
-#### Examples to access the defaultClient of your apolloProvider
+### Example with Vue composition API
+#### Provide client
+```ts
+// plugins/apollo-provide-client.ts
+import { provide, reactive } from "nuxt-composition-api";
+import { Context, Plugin } from "@nuxt/types";
+import { ApolloClients, DefaultApolloClient } from "@vue/apollo-composable";
+
+const provideApollo: Plugin = ({ app }: Context) => {
+  app.setup = () => {
+    const clients = reactive(app.apolloProvider?.clients);
+    provide(DefaultApolloClient, clients?.defaultClient);
+    return { [ApolloClients]: clients }; // not req'd but need to return data
```

Comment:
This functionally does nothing, but is required to conform to app.setup's typings.

Reply:
I'm still uncomfortable with assigning `app.setup = () => { ... }` in a plugin as this will break as soon as another plugin tries to do the same thing.

See [this issue](https://github.com/nuxt-community/composition-api/issues/104) in the nuxt-composition-api repository for a discussion on this.